openmc/filter.py

```diff
@@ -205,6 +207,10 @@ def bins(self, bins):
     def num_bins(self):
         return len(self.bins)
 
+    @property
+    def shape(self):
+        return (self.num_bins,)
```

Comment:
```suggestion
        return (self.num_bins)
```
Just wondering if the comma is needed